sdcard/rootfs/etc/rc.local

```diff
@@ -10,5 +10,7 @@
 # bits.
 #
 # By default this script does nothing.
+# Powersave (Disable HDMI) ~30ma
+/opt/vc/bin/tvservice -o
```

Comment:
```suggestion
# check if hdmi is connected, otherwise turn off
if ! /opt/vc/bin/tvservice -s | grep  HDMI; then
  /opt/vc/bin/tvservice -o
fi
```

Reply:
good idea @dadav I tested on mine and it works perfectly